### DISCUSSION TITLE
Question about monai losses - squared pred

### DISCUSSION THREAD

**Discussion by 25benjaminli** [Project-MONAI/tutorials#1638] [General]
I'm trying to understand when you would prefer using squared_pred in monai losses but haven't found anything on this subject.

**Comment by ericspod** [comment]
I would think that much like MSE this emphasises the differences and gives less weight to correct values. Which tutorial are you looking at? It may be that the original contributor had more of a rationale so you could reference them here.

**Reply by 25benjaminli** [reply]
Thanks for the response @ericspod. 

As an example, the swin unetr file uses squared_pred = False, but the brats_segmentation_3d file uses squared_pred=True. Links below. 

https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_brats21_segmentation_3d.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb

It seems like the origin for squared dice coefficient losses is https://arxiv.org/pdf/1606.04797.pdf (V-Net paper). It can also be seen in the SegResNet paper and some other papers. However, I can't seem to find any rationale for it. Your help is greatly appreciated!

**Reply by ericspod** [reply]
On the brats tutorial the last modification to the loss function line was by @Nic-Ma: would you have any insights here? My feeling is that it's an empirical thing where different people have gotten better results with that option on or off. You might have to just try both and see what works better.
